src/renault_api/kamereon/models.py

```diff
@@ -198,11 +198,23 @@ class EndpointDefinition:
     "XBG1VE": {  # DACIA SPRING
         "actions/charge-start": _KCM_ENDPOINTS["actions/charge-start"],
         "actions/charge-stop": _KCM_ENDPOINTS["actions/charge-stop"],
+        "alerts": None,  # Reason: "err.func.wired.not-found"
         "battery-status": _DEFAULT_ENDPOINTS["battery-status"],
-        "charge-mode": None,
+        "charge-history": None, # Reason: "err.func.wired.not-found"
+        "charge-mode": None, # Reason: "err.func.wired.forbidden"
+        "charge-schedule": None, # Reason: "err.func.wired.forbidden"
+        "charging-settings": None, # Reason: "err.func.wired.forbidden"
+        "cockpit": _DEFAULT_ENDPOINTS["cockpit"],
+        "hvac-history": None,  # Reason: "err.func.wired.not-found"
+        "hvac-sessions": None,  # Reason: "err.func.wired.not-found"
+        "hvac-settings": None,  # Reason: "err.tech.vcps.ev.hvac-settings.error"
+        "hvac-status": _DEFAULT_ENDPOINTS["hvac-status"],
         "location": _DEFAULT_ENDPOINTS["location"],
-        "lock-status": None,
-        "res-state": None,
+        "lock-status": None, # Reason: "err.func.wired.notFound"
+        "notification-settings": None, # Reason: "err.func.vcps.users-helper.get-notification-settings.error"
```

Comment:
This one is too long, and need `# noqa: E501` at the end
```suggestion
        "notification-settings": None,  # Reason: "err.func.vcps.users-helper.get-notification-settings.error"  # noqa: E501
```